pypit/pypit.py

```diff
@@ -67,9 +72,10 @@ def PYPIT(redname, progname=__file__, quick=False, ncpus=1, verbose=1,
     ---------------------------------------------------
     """
     # Init logger
-    import armsgs
-    msgs = armsgs.get_logger((logname, debug, last_updated, version, verbose))
-    import arload
+    msgs = armsgs.get_logger((logname, imsgs._debug, imsgs._last_updated,
```

Comment:
I'm not sure imsgs is used after this line, and we only use some of the variables (i.e. not the class itself). So, I think it's best not to pass imsgs as an argument into PYPIT